frontend/src/components/Layout/Header/Header.styles.ts

```diff
@@ -21,8 +21,8 @@ export const useStyles = () => {
       flexDirection: "row",
       justifyContent: "space-between",
       alignItems: "center",
-      shadowOffset: { width: 0, height: 1 },
-      shadowOpacity: 0.16,
+      shadowOffset: { width: 0, height: 2 },
+      shadowOpacity: 0.05,
```

Comment:
shadowOffset, shadowOpacity, shadowRadius, etc. attributes only affect web and ios
elevation only affects android

in other words, this change is only updating the shadows for web and ios

i used this site: https://10015.io/tools/react-native-shadow-generator to match the shadows across all platforms

Reply:
nice 👍